### PR TITLE
Geolocation accuracy

### PR DESCRIPTION
**Pull request by maawoo**
Stage 1 implementation described in https://github.com/SAR-ARD/S1_NRB/issues/33

### CODE REVIEW

File: S1_NRB/metadata/extract.py [hunk 1]
```diff
@@ -427,6 +427,58 @@ def _get_block_offset(band):
...
+    with Raster(ei_tif) as ras:
+        stats = ras.allstats(approximate=False)
+        ei_min = stats[0]['min']
+    
+    # COP-DEM global mean accuracy (LE68) based on LE90 under assumption of gaussian distribution:
+    copdem_glob_1sigma_le68 = 1.56
+    rmse_dem_planar = copdem_glob_1sigma_le68 / math.tan(math.radians(ei_min))
+    
+    # Calculation of RMSE_planar
+    rmse_planar = round(math.sqrt(
```
Comment (johntruckenbrodt) [inline S1_NRB/metadata/extract.py:474]: Please make it a bit more verbose:

```python
rmse_rg = math.sqrt(slc_acc['ALE']['rg'] ** 2 + slc_acc['1sigma']['rg'] ** 2)
rmse_az = math.sqrt(slc_acc['ALE']['az'] ** 2 + slc_acc['1sigma']['az'] ** 2)

rmse_planar = math.sqrt((rmse_rg / math.sin(math.radians(ei_min))) ** 2 + 
                        rmse_az ** 2  + 
                        rmse_dem_planar ** 2)
```

**Review by johntruckenbrodt — COMMENTED** [review]
Looks good. Only a minor change for improved readability.

File: S1_NRB/metadata/extract.py (hunk 1 above)
Comment (maawoo) [reply S1_NRB/metadata/extract.py]: Done

File: S1_NRB/metadata/extract.py [hunk 2]
```diff
@@ -490,6 +545,9 @@ def meta_dict(config, target, src_ids, snap_datasets, proc_time, start, stop, co
     tups = [(ITEM_MAP[key]['suffix'], ITEM_MAP[key]['z_error']) for key in ITEM_MAP.keys()]
     z_err_dict = dict(tups)
     
+    geocorr_acc = calc_geolocation_accuracy(swath_identifier=meta['common']['swathIdentifier'], ei_tif=ei_tif,
+                                            dem_type=dem_type, etad=config['etad'])
+
```
Comment (johntruckenbrodt) [inline S1_NRB/metadata/extract.py:550]: `meta['common']['swathIdentifier']` is queried but does not exist before line 575.

**Review by johntruckenbrodt — CHANGES_REQUESTED** [review]
got another one ;-)

**Pull request merged**